tests/unit/example/test.spec.js

```diff
@@ -0,0 +1,9 @@
+var assert = require('assert');
+
+describe('Array', function () {
+	describe('#indexOf()', function () {
```

Comment:
Why the "describe" is  nested in another "describe" ?

Reply:
This is just an example. And yes, `describe`s in `describe`s are valid mocha specs. 

It is mostly used to separate `it` (test cases).
Also, `describe`s and `context`s have own lifecycle hooks - therefore if you need to do separate action before 3 of your tests, but not before the fourth one, you'll wrap the three into the `describe`/`context` and define `beforeEach` hook in that nested `describe`/`context`.